wgengine/router/router_linux.go

```diff
@@ -1232,6 +1234,34 @@ var ipRules = []netlink.Rule{
 	// usual rules (pref 32766 and 32767, ie. main and default).
 }
 
+// udmProIPRules are the policy routing rules that Tailscale uses, when running
+// on a UDM-Pro.
+//
+// The priority is the value represented here added to
+// r.ipPolicyPrefBase, which is usually 5200.
+//
+// This represents an experiment that will be used to gather more information.
+// If this goes well, Tailscale may opt to use this for all of Linux.
+var udmProIPRules = []netlink.Rule{
+	// non-fwmark packets fall through to the usual rules (pref 32766 and 32767,
```

Comment:
Can you include what ip rules are seen on UDM Pro?

On a USG4 with failover configured, I see:
```
$ ip rule show
0:	from all lookup local 
201:	from all fwmark 0x64800000/0x7f800000 lookup 201 
202:	from all fwmark 0x65000000/0x7f800000 lookup 202 
32766:	from all lookup main 
32767:	from all lookup default
```

Reply:
@sailorfrag I'm not sure what you're asking. This is the result from my UDM Pro with and without tailscale running with this code:

```text
root@UDM-Pro:~# systemctl stop tailscaled
root@UDM-Pro:~# ip rule show
0:      from all lookup local
32000:  from all lookup main
32501:  from all fwmark 0x1c0000/0x7e0000 lookup 202.eth9
32504:  from 98.50.44.21 lookup 202.eth9
32765:  from all fwmark 0x10000/0x10000 lookup 251.blackhole
32766:  from all lookup 202.eth9
32767:  from all lookup default
root@UDM-Pro:~# systemctl start tailscaled
root@UDM-Pro:~# ip rule show
0:      from all lookup local
5270:   not from all fwmark 0x80000/0xff0000 lookup 52
32000:  from all lookup main
32501:  from all fwmark 0x1c0000/0x7e0000 lookup 202.eth9
32504:  from 98.50.44.21 lookup 202.eth9
32765:  from all fwmark 0x10000/0x10000 lookup 251.blackhole
32766:  from all lookup 202.eth9
32767:  from all lookup default
```